plutus-benchmark/nofib/test/Spec.hs

```diff
@@ -41,9 +42,9 @@ testClausify = testGroup "clausify"
                , testCase "formula3" $ mkClausifyTest Clausify.F3
                , testCase "formula4" $ mkClausifyTest Clausify.F4
                , testCase "formula5" $ mkClausifyTest Clausify.F5
+               , Tx.fitsInto "formula1 (size)" (Clausify.mkClausifyCode Clausify.F1) 5190
```

Comment:
I see that you're just testing the size and budget for one input to each program.  The size (and very much the budget) will depend on the input, so should we test all of the inputs for size and cost?

Reply:
Yeah, although I'm not sure how interesting it is. Maybe it's interesting enough. The reason I didn't want to add too many is that the way these work at the moment the threshold is written in the code, so they're slightly annoying to update.